lib/ProductOpener/Orgs.pm

```diff
@@ -160,9 +162,33 @@ sub store_org ($org_ref) {
 	# retrieve eventual previous values
 	my $previous_org_ref = retrieve("$BASE_DIRS{ORGS}/" . $org_ref->{org_id} . ".sto");
 
-	if ((defined $previous_org_ref) && !$previous_org_ref->{validated} && $org_ref->{validated}) {
+	if ((defined $previous_org_ref) && $previous_org_ref->{valid_org} ne 'on' && $org_ref->{valid_org} eq 'on') {
 		# we switched on validated
-		# TODO: create org and its users in Odoo CRM
+
+		eval { 
+			my $crm_org_id = create_company($org_ref);
+			die "Failed to create company" 
+			if not defined $crm_org_id;
+
+			my $user_ref = retrieve_user($org_ref->{creator});
+			my $crm_user_id = create_contact($user_ref);
+			die "Failed to create contact" 
+			if not defined $crm_user_id;
+
+			die "Failed to add contact to company" 
+			if not defined add_contact_to_company($crm_user_id, $crm_org_id);
+
+			die "Failed to create opportunity" 
+			if not defined create_opportunity($user_ref, $org_ref);
```

Comment:
Maybe store the opportunity id as well, so that we can update it later, for instance if the org imports products.